src/extract.rs

```diff
@@ -204,6 +203,9 @@ mod test {
 
     #[test]
     fn test_file_type() {
+        // Assume HTML in case there is no extension (that's including URLs
+        // without a path)
```

Comment:
What about plain text UNIX files? I know this is being picky, but something like plain-text README would be unnecessarily parsed as HTML.

How about this: default to HTML for URLs and to plaintext for paths/files?

Reply:
You're right. Fixed that now.